packages/create-hint/src/templates/meta.ts.hbs

```diff
@@ -34,8 +34,8 @@ const meta: HintMetadata = {
         /*
          * If you want to allow the user to configure your hint
          * you should use a valid JSON schema. More info in:
-         * https://webhint.io/docs/contributor-guide/hints/#themetaproperty
-         */
+         * https://webhint.io/docs/contributor-guide/how-to/hint/#the-meta-property
+         *
```

Comment:
you need to close the comment :)

Reply:
ah, thanks @sarvaje! accidentally deleted a line 😅